frontend/packages/console-shared/src/components/markdown-highlight-extension/MarkdownHighlightExtension.tsx

```diff
@@ -32,6 +32,6 @@ const MarkdownHighlightExtension: React.FC<MarkdownHighlightExtensionProps> = ({
   if (!selector) {
     return null;
   }
-  return <Spotlight selector={selector} interactive />;
+  return <Spotlight selector={selector} interactive checkHidden />;
```

Comment:
Why do you need to have this prop? You can simple do the check everytime.

Reply:
updated